app/renderer/components/tabs/tab.js

```diff
@@ -301,13 +312,15 @@ class Tab extends React.Component {
         className={css(
           styles.tab,
           // Windows specific style
-          isWindows && styles.tabForWindows,
-          this.props.isPinnedTab && styles.isPinned,
-          this.props.isActive && styles.active,
-          this.props.isActive && this.props.themeColor && perPageStyles.themeColor,
+          isWindows && styles.tab_forWindows,
+          this.props.isPinnedTab && styles.tab_pinned,
+          this.props.isActive && styles.tab_active,
+          this.props.showAudioTopBorder && styles.tab_audioTopBorder,
+          this.props.isActive && this.props.themeColor && perPageStyles.tab_themeColor,
```

Comment:
would you mind moving this line up?

Reply:
@cezaraugusto I'm sorry if I'm wrong, but I thought these should be:

````
          this.props.isActive && styles.active,
          this.props.isActive && this.props.themeColor && perPageStyles.themeColor,
          this.props.isPlayIndicatorBreakpoint && this.props.canPlayAudio && styles.narrowViewPlayIndicator,
````

to group them based on `this.props.isActive`. wdyt?

Reply:
w00t lost in the rebase, let me re-do

Reply:
weird I'm seeing them as you want them to be

<img width="632" alt="screen shot 2017-09-15 at 1 24 37 pm" src="https://user-images.githubusercontent.com/4672033/30493127-61d1ac42-9a19-11e7-9f0d-38b9e98e138c.png">

Reply:
dohh I'm looking for the wrong line

Reply:
in this commit we have:

```js
            styles.tab,
            perPageGradient.tab_gradient,
            // Windows specific style
            isWindows && styles.tab_forWindows,
            this.props.isPinnedTab && styles.tab_pinned,
            this.props.isActive && styles.tab_active,
            this.props.isActive && this.props.themeColor && perPageStyles.tab_themeColor,
            this.props.showAudioTopBorder && styles.tab_audioTopBorder
```

do you think this can be improveed?

Reply:
LGTM! 👍